SwiftLeeds/Views/My Conference/SpeakerView.swift

```diff
@@ -80,6 +96,10 @@ struct SpeakerView: View {
         .sheet(isPresented: $showWebSheet) {
             WebView(url: presentation.slidoURL ?? "")
         }
+        .sheet(isPresented: $showVideo) {
+            WebView(url: presentation.videoURL ?? "")
+                .edgesIgnoringSafeArea(.bottom)
+        }
```

Comment:
I feel like we should openURL externally here, as it will open the webpage in YouTube (if installed) which will offer a better user experience. Thoughts?

Reply:
That was my original approach then changed to in-app but happy either way

Reply:
I'd be a lot more happy with opening in YouTube natively, personally.